### ISSUE TITLE
[aws_ecs_cluster] Perpetually updating `setting` block

### ISSUE THREAD

**Issue by tarfeef102** [hashicorp/terraform-provider-aws#20728]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

Terraform v. 1.0.4
AWS Provider Version 3.54.0

### Affected Resource(s)

* aws_ecs_cluster

### Terraform Configuration Files

```
resource "aws_ecs_cluster" "example" {
  name = "example"
  setting {
    name  = "containerInsights"
    value = "enabled"
  }
}
```

### Expected Behavior

This should apply once, and then terraform should be happy

### Actual Behavior

Every apply, terraform detects out-of-band changes like so
```
# aws_ecs_cluster.apps has been changed
  ~ resource "aws_ecs_cluster" "example" {
        id                 = "EXAPMLE_ARN"
        name               = "example"
        tags               = {}
        # (3 unchanged attributes hidden)

      - setting {
          - name  = "containerInsights" -> null
          - value = "enabled" -> null
        }
    }
```
then the plan+apply brings the setting back

**Comment by ewbankkit** [comment]
@tarfeef102 Thanks for raising this issue.
It has already been noticed in #20684. I'm going to close this one as a duplicate so that we can concentrate discussion in the linked issue.
Please add any additional comments there.

The fix will be available in **v3.57.0** of the Terraform AWS Provider, likely later this week.

**Comment by tarfeef102** [comment]
> @tarfeef102 Thanks for raising this issue.
> It has already been noticed in #20684. I'm going to close this one as a duplicate so that we can concentrate discussion in the linked issue.
> Please add any additional comments there.
> 
> The fix will be available in **v3.57.0** of the Terraform AWS Provider, likely later this week.

thanks. didn't see that in my search because it was closed.
